src/__tests__/resources/events.integration.spec.ts

```diff
@@ -192,6 +192,17 @@ describe('Events integration test', () => {
       expect(items.length).toBeGreaterThan(0);
       expect(items[0].endTime).toBeUndefined();
     });
+
+    test('activeAtTime', async () => {
+      const { items } = await client.events.list({
+        filter: {
+          activeAtTime: { min: 101 },
+        },
+        limit: 10,
+      });
+
+      expect(items.length).toBe(1);
```

Comment:
what if somebody adds an event to the tenant which will be at the same time range?
is it better to use `toBeGreaterThan(0)`?

Reply:
What about https://github.com/cognitedata/cognite-sdk-js/pull/365/commits/fa6802afca6a8e5962708b190adfcbf67d972b03 ?

Reply:
yep, a much better solution

Reply:
I reverted back. Test failed due to eventual consistency